src/index.js

```diff
@@ -253,11 +253,11 @@ class DatastorePubsub extends Adapter {
   /**
    * Select the best record according to the received select function
    *
-   * @param {Uint8Array} receivedRecord
-   * @param {Uint8Array} currentRecord
+   * @param {Uint8Array} key
+   * @param {Uint8Array[]} records
    */
-  async _selectRecord (receivedRecord, currentRecord) {
-    const res = await this._validator.select(receivedRecord, currentRecord)
+  async _selectRecord (key, records) {
+    const res = await this._validator.select(key, records)
```

Comment:
This was wrong before!? 😱 

https://github.com/libp2p/js-libp2p-kad-dht/blob/master/src/content-fetching/index.js#L154
https://github.com/libp2p/js-libp2p-record/blob/master/src/selectors/public-key.js#L11